editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -4259,67 +4259,28 @@ void SpatialEditor::_init_indicators() {
 		indicator_mat->set_flag(SpatialMaterial::FLAG_ALBEDO_FROM_VERTEX_COLOR, true);
 		indicator_mat->set_flag(SpatialMaterial::FLAG_SRGB_VERTEX_COLOR, true);
 
-		PoolVector<Color> grid_colors[3];
-		PoolVector<Vector3> grid_points[3];
 		Vector<Color> origin_colors;
 		Vector<Vector3> origin_points;
 
-		Color grid_color = EditorSettings::get_singleton()->get("editors/3d/grid_color");
+
```

Comment:
This causes style issues for `clang-format`: https://travis-ci.org/godotengine/godot/jobs/313726772 (it's configured to only allow one empty line between statements).